docker-bake.hcl

```diff
@@ -0,0 +1,39 @@
+variable "GO_VERSION" {
+  default = "1.18"
+}
```

Comment:
This also defined in the Dockerfile. We need an explanation comment why we need this duplication.

Reply:
It's here because from now on we manage containerization process through bake command, so, if we want to override the Go version, we'll be overriding it through the bake command using a syntax like "--set <target>.args.GO_VERSION=<version>"